crane-cloud/backend
# Add App Revision feature

# Description
This PR adds app revisions list and ability to roll back to specific revision
## Type of change

Please delete options that are not relevant.



## How Can This Be Tested?

Go to the App details route (**GET** - `/apps/{app_id}`), and you should be able to see the app revisions list.
A new route has been added, **POST**  - `/apps/{:app_id}/revise/{:revision_id}`, here you should be able to pass a `revision_id` and we revert the application to that `ids` instance


## Checklist:
